index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="index.css">
+    <title>NEWSLETTER</title>
+</head>
+<body>
+    <section class="section1">
+        <div class="image">
+            <img src="/newsletter/assets/images/illustration-sign-up-mobile.svg" alt="">
+        </div>
+        <div class="writting">
+            <h1>Stay updated!</h1>
+            <h4>Join 60,000+ product managers receiving monthly updates on:
+                <ul>
+                    <li><img src="/newsletter/assets/images/icon-list.svg" alt=""> Product discovery and building what matters</li>
+                    <li><img src="/newsletter/assets/images/icon-list.svg" alt=""> Measuring to ensure updates are a success</li>
+                    <li><img src="/newsletter/assets/images/icon-list.svg" alt=""> And much more!</li>
+                </ul>
+            </h4>
```

Comment:
🎲  I think this </h4> end tag is  misplaced as it wraps up the entire <ul>.....</ul>